caddyhttp/index/index_test.go

```diff
@@ -17,8 +17,10 @@ package index
 import (
 	"testing"
 
-	"github.com/mholt/caddy"
 	"github.com/mholt/caddy/caddyhttp/staticfiles"
+
+	"github.com/mholt/caddy"
+	"github.com/mholt/caddy/caddyhttp/httpserver"
 )
```

Comment:
Delete whitespace and gofmt to rearrange imports

```
import (
        "testing"

        "github.com/mholt/caddy"
        "github.com/mholt/caddy/caddyhttp/httpserver"
        "github.com/mholt/caddy/caddyhttp/staticfiles"
)

```

Reply:
Done